resources/views/frontend/catalog-new/index-new.blade.php

```diff
@@ -212,7 +212,15 @@ class="tw-font-bold">(@{{ artworks.total }})</span>
                         :selected-options-as-labels="selectedOptionsAsLabels" :selected-values="query">
                     </filter-new-selected-labels>
                 </div>
-                <div class="tw-px-4 md:tw-p-16">
+                <div v-if="artworks.length === 0"
+                    class="tw-flex tw-w-full tw-flex-col tw-items-center tw-justify-center tw-py-40 tw-text-lg">
+                    <div class="tw-w-72">
+                        <empty-artworks-animation></empty-artworks-animation>
```

Comment:
I'd extract this out of the component and just use the `<lottie-player>` component directly, passing a URL instead of an object
```suggestion
                        <lottie-player src={{ asset('animations/empty.json') }}></lottie-player>
```

That way the animation doesn't have to become part of the JS bundle (and it's less code)

See https://docs.lottiefiles.com/lottie-player/components/lottie-player/usage#with-reactjs-and-vuejs